src/commands/service/create.ts

```diff
@@ -27,14 +27,24 @@ export default class ServiceCreate extends Command {
     const {args, flags} = this.parse(ServiceCreate)
     const credential = await this.getCredential()
     this.spinner.start('Create service')
-    const resp = await this.api.service.create(JSON.parse(args.DEFINITION), credential)
-    if (!resp.hash) { throw new Error('invalid response') }
-    this.spinner.stop(base58.encode(resp.hash))
+    const service = JSON.parse(args.DEFINITION)
+
+    const {hash} = await this.api.service.hash(service)
+    if (!hash) throw new Error('invalid hash')
+    const {exists} = await this.api.service.exists({hash})
+    if (exists) {
+      this.warn('Service already exists')
```

Comment:
why not checking if service exist and not simply output the error from service.create?

Reply:
done